conda_build/post.py

```diff
@@ -427,6 +429,11 @@ def mk_relative_linux(f, prefix, rpaths=('lib',)):
     except CalledProcessError:
         print('patchelf: --print-rpath failed for %s\n' % (elf))
         return
+    print("existing paths via patchelf: {}".format(existing))
```

Comment:
This looks like debugging output.  Can it be done with log.debug instead of print?